torch/nested/_internal/nested_tensor.py

```diff
@@ -76,31 +98,17 @@ def __new__(
             False,
             True,  # dispatch_layout
             ks,
+            # don't try to calculate storage based on non-zero size
+            _override_storage_size=values.numel() * values.dtype.itemsize,
```

Comment:
Not sure where this matters tbh, but what about `values.untyped_storage().size()`?